README.md

```diff
@@ -7,8 +7,15 @@
 This repo is the canonical source for Kubernetes Operators that appear on [OperatorHub.io](https://operatorhub.io), [OpenShift Container Platform](https://openshift.com) and [OKD](https://okd.io).
 
 ## Documentation
+
 Full documentation is generated via [mkdoc](https://www.mkdocs.org/) and is located at [https://operator-framework.github.io/community-operators/](https://operator-framework.github.io/community-operators/)
 
+## IMPORTANT NOTICE
```

Comment:
```suggestion
## IMPORTANT NOTICE FOR OPENSHIFT COMMUNITY OPERATORS
```

Reply:
HI @dmesser the following msg about deprecated apis are valid for Kubernetes as well. It is not specific for OCP. So, do you think that change title would make sense?

Reply:
done as we spoke about